Lib/test/libregrtest/runtests.py

```diff
@@ -93,7 +93,7 @@ class RunTests:
     python_cmd: tuple[str, ...] | None
     randomize: bool
     random_seed: int | str
-    json_file: JsonFile | None
+    json_file: JsonFile
```

Comment:
`Regrtest.create_run_tests()` creates RunTests() with json_file=None. This change looks wrong.